lib/cron-job/cron-job-props.ts

```diff
@@ -38,4 +38,14 @@ export interface CronJobProps
    * Specifies the deadline in seconds for starting the job if it misses its scheduled time.
    */
   readonly startingDeadlineSeconds?: number;
+
+  /**
+   * Specifies the number of successful finished jobs to retain. @default 3
```

Comment:
`@` tags must be on their own lines to be picked up correctly in IDEs (you should see it in VSCode when you instantiate an object and hover over the prop)